include/fiction/technology/sidb_on_the_fly_gate_library.hpp

```diff
@@ -99,9 +98,9 @@ class gate_design_exception : public std::exception
 /**
  * This struct encapsulates parameters for the parameterized SiDB gate library.
  *
- * @tparam Lyt SiDB cell-level layout type.
+ * @tparam CellType Cell type.
```

Comment:
SiDB cell type?

Are `static_assert`s for that in place?

Reply:
Good point! Thanks